src/libsystemd/sd-json/sd-json.c

```diff
@@ -76,7 +76,11 @@ struct sd_json_variant {
         /* The current 'depth' of the sd_json_variant, i.e. how many levels of member variants this has */
         uint16_t depth;
 
-        int type:8; /* actually sd_json_variant_type_t, but reduced to 8bit size */
+        signed int type:8; /* actually sd_json_variant_type_t, but reduced to 8bit size (we used "signed int"
+                            * rather than just "int", since apparently compilers are free to pick signedness
+                            * otherwise. We don't really care about the signendess, we care more about
```

Comment:
```suggestion
                            * otherwise. We don't really care about the signeess, we care more about
```

```suggestion
                            * otherwise. We don't really care about the signedness, we care more about
```